src/components/quiz/GameOver.tsx

```diff
@@ -0,0 +1,22 @@
+interface GameOverProps {
+  resetGame(): void;
+}
+
+function GameOver({ resetGame }: GameOverProps) {
+  return (
+    <div className="QuizPage__result">
+      <h2>Game Over: Aliens have taken over earth</h2>
```

Comment:
Give it a bit of pizzazz! 😉 
```suggestion
      <h2>Game Over: Aliens have taken over earth!! 😱👽</h2>
```